docs/analysis-workflows/harmonization-dna-pipeline.md

```diff
@@ -23,39 +23,81 @@ The ARGO Data Platform accepts raw sequencing data in both FASTQ and BAM (aligne
 
 ### Inputs
 
-- All alignments are performed using the human reference genome **[GRCh38](http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/reference/GRCh38_reference_genome)** as the reference genome
+- All alignments are performed using the human reference genome [GRCh38](http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/reference/GRCh38_reference_genome) as the reference genome
 - Submitted FASTQ or BAM files(s)
 
 ### Preprocessing
 
 - Submitted sequencing reads (FASTQ or BAM) are converted into lane level (i.e read group level) BAMs
-- Use Picard tool `CollectQualityYieldMetrics` for read group QC
+- `Picard CollectQualityYieldMetrics` is used for read group level BAM QC
 
 ### Processing
 
-- For each lane BAM **[BWA-MEM version 0.7.17-r1188](https://github.com/lh3/bwa/archive/v0.7.17.tar.gz)** is run.
-- Once all lane BAMS have completed, **[Biobambam version 2.0.153](https://gitlab.com/german.tischler/biobambam2/-/archive/2.0.153-release-20200124123734/biobambam2-2.0.153-release-20200124123734.tar.gz)** is used to merge lane BAMs and mark duplicates.
-- Calculate Alignment QC metrics using `samtools stats`
-- Calculate `OxoQ` score to assess oxidative artifacts is calculated with Picard `CollectOxoGMetrics`
+- For each read group, [BWA-MEM (version 0.7.17-r1188)](https://github.com/lh3/bwa/archive/v0.7.17.tar.gz) is performed to map the reads to the reference genome.
+- For each sample, [Biobambam (version 2.0.153)](https://gitlab.com/german.tischler/biobambam2/-/archive/2.0.153-release-20200124123734/biobambam2-2.0.153-release-20200124123734.tar.gz) is used to merge all read group level mapped BAMs into sample level BAM and mark duplicates per library.
+- `Samtools stats` is used to calculate Alignment QC metrics
+- `Picard CollectOxoGMetrics` is used to calculate `OxoQ` score to assess oxidative artifacts 
 
 ### Outputs
 
-- Aligned Read CRAM file
-- CRAI Index file
-- Alignment QC Metrics
+- Aligned read CRAM and index files
+- Alignment QC metrics files

```

Comment:
maybe adding detailed output qc_metrics file types? like alignment qc, oxog metrics etc

Reply:
Christina suggested a page listing all file types, with the detailed description of each files in a separate page.

Reply:
Okay, so we will leave the `Alignment QC metrics files` as it is here.